.github/workflows/unittests.yml

```diff
@@ -116,24 +53,31 @@ jobs:
         run: ./bazelisk run larq_compute_engine/tflite/tests:cc_tests_aarch64_qemu --config=aarch64
 
   MLIR:
-    runs-on: ubuntu-latest
-    container:
-      image: tensorflow/tensorflow:custom-op-ubuntu16
-
+    runs-on: macos-latest
     steps:
       - uses: actions/checkout@v1
       - name: Install Bazelisk
         run: |
-          curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.3.0/bazelisk-linux-amd64 > bazelisk
+          curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.3.0/bazelisk-darwin-amd64 > bazelisk
           chmod +x bazelisk
-      - name: Force Python 3 for everything we do
-        run: |
-          ln -sf /usr/bin/python3.6 /usr/bin/python
-          ln -sf /usr/bin/python3.6 /usr/bin/python3
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
       - name: Configure Bazel
-        run: ./configure.sh <<< $'yes\nyes\n' #yes,yes for tf2
-        shell: bash
+        run: ./configure.sh
       - name: Install pip dependencies
-        run: python -m pip install larq_zoo==0.5.0 pytest --no-cache
-      - name: Build LCE MLIR passes
-        run: ./bazelisk run larq_compute_engine/mlir:converter_test
+        run: pip install tensorflow==2.1.0 larq_zoo==0.5.0 pytest tensorflow_datasets==1.3.2 --no-cache
+      - name: Run converter tests TF 2.1
+        run: ./bazelisk run larq_compute_engine/mlir:converter_test --distinct_host_configuration=false
+      - name: Install TF 2.0
+        run: pip install tensorflow==2.0.0 --no-cache
+      - name: Run converter tests TF 2.0
+        run: ./bazelisk run larq_compute_engine/mlir:converter_test --distinct_host_configuration=false
+      - name: Install TF 1.15
+        run: pip install tensorflow==1.15 --no-cache
+      - name: Run converter tests TF 1.15
+        run: ./bazelisk run larq_compute_engine/mlir:converter_test --distinct_host_configuration=false
+      - name: Install TF 1.14
+        run: pip install tensorflow==1.14 --no-cache
+      - name: Run converter tests TF 1.14
+        run: ./bazelisk run larq_compute_engine/mlir:converter_test --distinct_host_configuration=false
```

Comment:
We could parallelize this in the future, though then we'd need to build a wheel first and store it in an artifact.

Reply:
I think this is fine.
However, after installing a different tensorflow version, technically you should run `./configure.sh` again because it will update some TF paths. I think its fine in this case because all versions are installed to the same path here.

However, for TF 1.14 especially, we should run it again but with `no` to the manylinux2010 question.

Reply:
> ./configure.sh again because it will update some TF paths. I think its fine in this case because all versions are installed to the same path here.

I removed the path configs from the ./configure.sh script since they are only needed when building custom ops that dynamically link against TensorFlow